py3status/modules/speedtest.py

```diff
@@ -0,0 +1,273 @@
+# -*- coding: utf-8 -*-
+"""
+Do a bandwidth test with speedtest-cli.
+
+Configuration parameters:
+    button_share: button to open the result link (default None)
+    format: display format for this module, {download} and/or {upload} required
+        *(default 'Speedtest [\?color=darkgray ping [\?color=ping {ping} ms] '
+                    'up [\?color=upload {upload} {upload_unit}] '
+                    'down [\?color=download {download} {download_unit}]]')*
+    server_id: speedtest server to use, `speedtest-cli --list` to get id (default None)
+    si_units: use SI units (default False)
+    thresholds: specify color thresholds to use *(default {
+                'download': [(0, 'bad'), (1024, 'degraded'), (1024 * 1024, 'good')],
+                'ping': [(200, 'bad'), (150, 'orange'), (100, 'degraded'), (10, 'good')],
+                'upload': [(0, 'bad'), (1024, 'degraded'), (1024 * 1024, 'good')]})*
```

Comment:
(Qeustion) You have sources for this numbers? How do you know what's good, degraded, bad, etc?

Reply:
Maybe... Users have different speeds and needs. I don't want to keep painting things red for some users. Alternatively, we could paint them all with a color that is similar to speedtest logo/theme... kinda like arbitrary `loadavg` blue... Maybe try `#6AFFF3`... Not sure how it looks (color is from speedtest.net's stars). Hope it's not too bright like the stars. :)